packages/styled-components/components/socialSharing/socialSharing.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import {
+  PureSocialSharing as SocialSharing,
+  SupportedPlatforms,
+} from '.';
+
+export default {
+  component: SocialSharing,
+  title: 'Irving/SocialSharing',
+};
+
+// eslint-next-line max-len
```

Comment:
Does this work? I thought it was 
```suggestion
// eslint-disable-next-line max-len
```
Is it necessary? line 15 looks even longer.

Reply:
I guess I assumed it'd be an issue, but the linter is quiet. Removed.